langchain/document_loaders/csv_loader.py

```diff
@@ -30,19 +30,13 @@ def __init__(
         self,
         file_path: str,
         source_column: Optional[str] = None,
-        csv_args: Optional[Dict] = None,
+        csv_args: Optional[Dict] = {},
```

Comment:
should keep as None, mutable objects as default values cause problems (https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)

Reply:
TIL. Changed it back to none